p2panda-js/test/sample-values.ts

```diff
@@ -0,0 +1,17 @@
+/**
+ * These sample values can be used in tests. They match example values in the
+ * openrpc spec, which is used to drive the mock openrpc server.
+ */
+export default {
+  // PRIVATE_KEY: '4c21b14046f284f87f1ea4be4b973664221ad483079a68ed35a6812553b41176',
```

Comment:
Why is this commented out?

Reply:
My guess is that it's not used atm, but is useful to have if we want to generate more test values later on.

Reply:
Then I would just uncomment it 👍